.github/workflows/regular.yml

```diff
@@ -1,7 +1,8 @@
 # NOTES:
 # - The map syntax used for matrix is flagged red but actually works
 # - This runs everything in Python 3.10, 3.11 and 3.12
-# - No environments are cached due to space limit
+# - Environments are **not** cached in order to perform a full end-to-end test
```

Comment:
I'm confused, are environments cached ot not? The action looks like we cache stuff, can you elaborate?

Reply:
yep, we dont need caching here in regular

Reply:
Thanks, good catch. I reverted @Scienfitz commit but didn't pay enough attention (it also reactivated caching in `regular.yml`, which I did not intend). Is now fixed 👍🏼